timetracker/entries/forms.py

```diff
@@ -4,13 +4,18 @@
 from .models import Project, Client
 
 
-class ClientForm(forms.Form):
-    name = forms.CharField()
-
-
-class ProjectForm(forms.Form):
-	name = forms.CharField()
-	client = forms.ModelChoiceField(queryset=Client.objects.all())
+class ClientForm(forms.ModelForm):
+    class Meta:
+        model = Client
+        fields = '__all__'
```

Comment:
Yes, this is possible but I HARDLY recommend against it! Please be explicit in which fields you want on there. That way users might not suddenly be able to change data in fields they are not supposed to change.